### PR TITLE
Adds new Vulpkanin suit sprites

### PR DESCRIPTION
**Pull request by Xantholne**
Adds new vulpkanin sprites for ERT suits, Captain suit, Nasa voidsuit, SWAT armor, Paramedic, wizard armor, and several syndicate softsuits available to players.

5252ef6660a4a4188feee52576df830f
694a69498596af1046de5cc822fd36b0
72131ae7879fda5334aceb82e7e973cd

🆑 Xantholne
add: After many years worth of expenses and many campaigns later, Nanotrasen has finally made it so Vulpkanin crewmembers no longer have their tails, part of their head, and ears magically disappear from many commonly found softsuits and hardsuits.
/🆑

### CODE REVIEW

**Comment by Trubuss** [comment]
My only question is will these sprites mean we will have vulp ERT and Nuke Ops? So far they are just humans.

**Comment by LightFire53** [comment]
Vulp ERT would be admin bus. I don't think there are plans to be able to be an "alien" ERT.

**Comment by Xantholne** [comment]
Vulp ERT is just admin bus for now. It's completely possible that a non human ERT could happen in the future since it is something that can be easily coded in as a selection. There's more species that require sprites for that until then.

**Comment by Fox-McCloud** [comment]
The Nasa suit and paramedic one look pretty bad still; others seem ok.

**Comment by Xantholne** [comment]
Redoing nassa void suit entirely because i fucking hate it.

**Comment by SmokingKilz** [comment]
Aren't NT immensely xenophobic in our lore? Was that changed? If it wasn't I don't see none-human ERT being a standard with NT response teams.

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -441,9 +441,10 @@
 	heat_protection = HEAD												//Uncomment to enable firesuit protection
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 	unacidable = 1
-	species_fit = list("Grey")
+	species_fit = list("Grey","Vulpkanin")
```
Comment (SamHPurp) [inline code/modules/clothing/spacesuits/hardsuit.dm]: fix the spacing please

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -8,10 +8,11 @@
 	permeability_coefficient = 0.01
 	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)
 	species_restricted = list("exclude", "Diona", "Wryn")
-	species_fit = list("Vox", "Grey")
+	species_fit = list("Vox", "Grey","Vulpkanin")
```
Comment (SamHPurp) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: here too

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 3]
```diff
@@ -32,9 +33,10 @@
 	allowed = list(/obj/item/weapon/tank, /obj/item/device/flashlight,/obj/item/weapon/gun/energy, /obj/item/weapon/gun/projectile, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs)
 	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)
 	species_restricted = list("exclude", "Diona", "Wryn")
-	species_fit = list("Vox")
+	species_fit = list("Vox","Vulpkanin")
```
Comment (SamHPurp) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: and again

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 4]
```diff
@@ -137,10 +143,11 @@
 	desc = "A paramedic EVA helmet. Used in the recovery of bodies from space."
 	icon_state = "paramedic-eva-helmet"
 	item_state = "paramedic-eva-helmet"
-	species_fit = list("Vox", "Grey")
+	species_fit = list("Vox", "Grey","Vulpkanin")
```
Comment (SamHPurp) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: you betcha

**Comment by taukausanake** [comment]
> NT immensely xenophobic in our lore

As a server that in by all intents, whenever we do or don't mean or want it, furry I would think we'd have a slight non-xenophobic  Nanotrasen. That's not to say they would enlist aliums into their military ranks, mainly because they've had years training humans and have a standard. Also if any real-world mention can be made about special needs individuals, for lack of a better term, it would be in their interests to NOT enlist anything they are not already used to working with

**Comment by Spacemanspark** [comment]
I don't see the need to add ERT hardsuits to only vulp. Add ones to the other races (Ones that already have custom sprites, anyways) too, for consistency.

**Comment by Tayyyyyyy** [comment]
Does it really make sense for hardsuits to have little tail things? I feel like it makes sense for unathi cause they have "hard" tails, but, correct me if I'm wrong, since I don't know much about tails, wouldn't a vulp tail be soft and able to be tucked in somewhere? Seems a little awkward to have that thing sticking out behind you all the time.

**Comment by Xantholne** [comment]
I was planning on that in a differen't PR actually with the eventual goal of a species selection check when joining as ERT

**Comment by Ty-Omaha** [comment]
The paramedic, ERT medic and orange space suit looks blurry

**Comment by Regen1** [comment]
I don't think there's a reason to have anything "vulp" that's adminbus only, it's bloat to the codebase that won't be used, if admins want them they can vv edit them in

**Comment by Ty-Omaha** [comment]
Last time I checked (few months ago) if a vulp crew member puts on an ERT suit it will be invisible on their body but still there

**Comment by Vivalas** [comment]
Don't think this really makes visual sense, tuck your ears and tails and shit in the suit, shouldn't be incredibly hard from a anatomical standpoint. A giant (fabric?) pouch for your tail is a bit silly, and the stripes on the sides for the sideways sprites looks kinda bad.

**Comment by Xantholne** [comment]
Ty is correct in that if a Vulp or several other non human species put on an ERT suit it's 100% invisible which is considered a bug. Similar to the Flangellant robes currently being removed entirely from the bottom half and giving an error cult hood is also a bug and also being exploited.

**Comment by Citinited** [comment]
Vulp tails are huge, large enough that "tucking them in" wouldn't really be feasible. Also, being able to immediately see "Oh that person is a vulpkanin" is probably a good thing.

@Regen1 88 lines of added code and a handful of sprites is hardly bloat to the codebase.

**Comment by Regen1** [comment]
>Vulp tails are huge, large enough that "tucking them in" wouldn't really be feasible.

Realism doesn't matter, as proven time and time again.

>"Oh that person is a vulpkanin" is probably a good thing. 

Why is it a good thing? No other races has that, and we even hide races on examine if they wear a full hardsuit.

Just use the normal ERT sprites

**Comment by Citinited** [comment]
Realism *when compared to something outside of the SS13 universe / compared to real life / compared to other thing not in SS13* makes no sense. Things should be somewhat consistent in-game though!

Thoughts on why it's a good thing - it lets you narrow down a list of "unknowns" to maybe 15% of its usual size (which is still a large number but not so large that you have no way of telling who is who). The hide species on examine thing might be a relict of when we didn't have so many species, but sure it's inconsistent as it is right now. (Not sure if this PR is adding to the inconsistency, or removing the inconsistency by making things consistently inconsistent...)

**Comment by Vivalas** [comment]
Unknowns should be unknowns, I suppose Unathi breacher suits are the exception, but we shouldn't widen the exception.

**Comment by ktoma36** [comment]
Personally I just love seeing more diversity while playing, even if its something I only see once in a blue moon. Im also curious, any thoughts about adding deathsquad armor sprites for different species?

**Comment by taukausanake** [comment]
I think the real issue is that if you're going to have clothings that do not have any specie restrictions then you need to have proper fitting sprites for all species whenever or not they will ever wear something. If you don't and a time comes when a specie does put on an unfitted clothing it will be reported as an issue. Best to do everything properly the first time

**Comment by Regen1** [comment]
>any thoughts about adding deathsquad armor sprites for different species?

Yes, it's bloat, deathsquads are exclusively human.

**Comment by Xantholne** [comment]
Deathquads spawn exclusively human and don't have any Roleplay around them outside *AYE CENTRAL COMMAND GUY WE'LL KILL EM ALL" so no not really. But like Taukausanake said if someone does put any of them on as a non human it will be reported as an issue and have to be fixed. So I do not see a reason for complaining so much over them.

**Comment by MarcellusPye** [comment]
@Regen1 Have you seen the emote code? I don't think we're allowed to complain about bloat until that's cleaned.

**Comment by Spacemanspark** [comment]
That doesn't mean we need to add more.

**Comment by taukausanake** [comment]
Question: I'm noticing a single orange pixel on the top of almost each east icon. What is that?

**Comment by Xantholne** [comment]
That was from me when I took them forgetting to move the east facing sprites one pixel to the right. That's part of the vulps ear you're seeing which was fixed shortly after.

**Comment by MarcellusPye** [comment]
@Spacemanspark This fixes a current bug and could be used in future, most of the emote code can be replicated with ! or me, please explain to me exactly how a few sprites and 88 lines is unacceptable while lets say 500 lines of LITERALLY USELESS code is kept and expanded upon.

**Comment by taukausanake** [comment]
I think the only excuse for bloat is resource downloading. Not a very good reason, not when we get features with more additions than this PR

**Comment by Regen1** [comment]
>Have you seen the emote code? I don't think we're allowed to complain about bloat until that's cleaned.

So because of that I can add whatever I want? 300mb of music here we come

>I think the only excuse for bloat is resource downloading. Not a very good reason, not when we get features with more additions than this PR

PR's that bring more than a dozen suits that will almost never be used, mostly because ERT's are exclusively human

**Comment by ktoma36** [comment]
oooh what about the clown and mime Eva suits @Xaltonon :D

**Comment by Xantholne** [comment]
I had taken a look at the mime and clown suits, saw just how many frames they have each. Said no screw that.

**Comment by ktoma36** [comment]
Fair enough

**Comment by Tayyyyyyy** [comment]
Here's what [actual space suits for dogs](http://gizmodo.com/5969041/drop-everything-dog-space-suits-right-here/) looked like. There's no tail appendages installed into the suit. I love the sprite work but the tail seems really unnecessary.

**Comment by taukausanake** [comment]
@Tayyyyyyy Yeah but that's for Ian. And maybe Renault. And the full reason for having the tail suit is for maximum comfort ~and totally not fursona reasons~ as any human/humanoid would demand. Next thing you know we'll start stuffing birbmen into suits made for humans and cramming fishfolk heads into glass bowls.

Wow. I am all for being a xeno-racist now just to see how it goes :laughing:

**Comment by Fox-McCloud** [comment]
ever get around to touching these up?

**Comment by Xantholne** [comment]
Having a hard time making a void suit that isn't a horrible helmet. Should have an updated picture soon of them updated the Version X

**Comment by tigercat2000** [comment]
This PR has been marked stale, and must be updated according to the below requests by 2017-09-17 (Sunday, 17th of September) or it shall be closed:
 - Remove the ERT suit sprites entirely. They serve no purpose but to deanonymize vulpkanin trying to hide themselves as an ERT member- The sprite should be falling back to using the normal human sprite. The clipping with the head is an issue, given, but that should be fixed in other ways than making the problem worse.
 - Resolve SamHPurp's comments about code styling.
 - As Fox detailed, the issues and lack of fidelity with the Voidsuit and Paramedic suits are still an issue.

**Comment by HugoLuman** [comment]
@Citinited Examining revealing species was actually added only relatively recently, within the past year or so.

**Comment by mrturkeytoe** [comment]
I miss hardsuit species compatibility conversion kits.

#BringBackHardsuitSpeciesLimitations2017

**Comment by ktoma36** [comment]
Awww

**Comment by Aurorablade** [comment]
why?

**Comment by taukausanake** [comment]
I think something exploded

**Pull request closed without merging**